pc/pc.go

```diff
@@ -252,3 +253,14 @@ func GetSitesIntegrity() pio.ConfigFile {
 	c.SiteHmac = vaultMac
 	return c
 }
+
+// GenHexString will generate a random 32 character hex string.
+func GenHexString() (s string, err error) {
+	var b [16]byte
+	_, err = rand.Read(b[:])
+	if err != nil {
+		return
```

Comment:
It's better to avoid naked returns. Be explicit. It's more readable.

```Go
return "", err
```

Especially since this is pretty critical crypto functionality, readability is even more important.

Also see https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters.